cranelift/assembler/meta/src/dsl/format.rs

```diff
@@ -158,6 +193,13 @@ impl Location {
             rm8 | rm16 | rm32 | rm64 => OperandKind::RegMem(*self),
         }
     }
+
+    pub fn to_usize(&self) -> usize {
```

Comment:
Seems like we should rename this to something more descriptive because without looking at any other code I don't know what this is trying to do. Also, are we sure anything non-imm8 always returns 1?